src/database/migrations/0042-create-base_fields_copy_tasks.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE base_fields_copy_tasks (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  status task_status NOT NULL,
+  synchronization_url VARCHAR NOT NULL,
+  status_updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  FOREIGN KEY (created_by) REFERENCES users(keycloak_user_id),
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  created_by UUID NOT NULL DEFAULT system_keycloak_user_id()
```

Comment:
Likewise for the [`capitalisation.functions` rule](https://docs.sqlfluff.com/en/stable/reference/rules.html#rule-capitalisation.functions): currently we're using the default rule of `consistent`, but I think maybe it should be `lower`.